src/cosmoz-tab-card.js

```diff
@@ -19,8 +21,10 @@ Custom property                         | Description              | Default
 `--cosmoz-tab-card-content-line-height` | Card content line height | `initial`
 `--cosmoz-tab-card-content-padding`     | Card content padding     | `initial`
 */
-const CosmozTabCard = ({ heading }) =>
-	html` <style>
+
+const CosmozTabCard = ({ heading, collapsable, collapsed }) => {
+	const [isCollapsed, setCollapsed] = useState(collapsed);
```

Comment:
The convention for state is 'stateName, setStateName'. `isCollapsed` leads me to believe that it's the result of some check, not an actual state.

```suggestion
	const [collapsed, setCollapsed] = useState(collapsed);
```

Reply:
Convention-wise renamed. Thanks!